### ISSUE TITLE
Not a valid path name

### ISSUE THREAD

**Issue by Lod3** [PACKED-vzw/CultURIze#15]
using VAI-test2 as subfolder gives error " Not Valid Path Name"

**Comment by nvanderperren** [comment]
it seems that this is solvable by changing the regular expression in the file `Publishing.ts, line 41`

i.s.o.
```ts
const dirRegex = /^((\w)+)(((\/)(\w+))+)?$/;
```
it should be 
```ts
const dirRegex = /^([a-z0-9_-]+)(((\/)([a-z0-9_-]+))+)?$/;
```

e.g. I could create this folder/link with culturize after changing that line: http://ec2-18-191-252-182.us-east-2.compute.amazonaws.com/test-test/data/22

(not sure though if this leads to erroneous creation of PIDs)

**Comment by nvanderperren** [comment]
closed by #45
